adapters/syslog/syslog.go

```diff
@@ -76,7 +77,14 @@ type SyslogAdapter struct {
 
 func (a *SyslogAdapter) Stream(logstream chan *router.Message) {
 	for message := range logstream {
-		err := a.tmpl.Execute(a.conn, &SyslogMessage{message, a})
```

Comment:
More specifically, without this internal buffering, tcpdump shows me 39 UDP packets received from a short-lived container that outputs 3 lines of text, whereas with this fix applied, I only see 3 packets, and these are also processed correctly by rsyslog.